utils/config.ts

```diff
@@ -88,9 +93,20 @@ export const IConfig = t.intersection([
   EUCovidCertProfileQueueConfig
 ]);
 
+const DEFAULT_EMAIL_MODE_SWITCH_LIMIT_DATE = new Date("1970-01-01T00:00:00Z");
+
 // No need to re-evaluate this object for each call
 const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   ...process.env,
+  EMAIL_MODE_SWITCH_LIMIT_DATE: fromNullableE(
```

Comment:
This value should be a string because [here](https://github.com/pagopa/io-functions-app/pull/174/files#diff-8f4625a1cf4cb245a2f9e780a9a17ff6a7132b8e070997144a67499d7ff793b4R99) we apply the decoder IConfig and 
UTCISODateFromString requires a string as input.